src/components/character-count/character-count.test.js

```diff
@@ -0,0 +1,400 @@
+/**
+ * @jest-environment ./lib/puppeteer/environment.js
```

Comment:
@timsb This is to keep testing aligned with `govuk-frontend`, otherwise we need to start re-writing test structure and diverging. Additionally, we do have a few other tests using Puppeteer, so although each additional test using Puppeteer increases the runtime, it is a standard within the library already, and I'd prefer this to stay aligned if possible.

Reply:
Ah ok. Hadn't reaslised govuk-frontend was using puppeteer as well. Couldn't see that comment in their version. Thought they might just be using jest-dom.